src/CoreShop/Bundle/TaxationBundle/DependencyInjection/Configuration.php

```diff
@@ -195,9 +195,11 @@ private function addPimcoreResourcesSection(ArrayNodeDefinition $node)
                             ->scalarNode('tax_item')->defaultValue('/bundles/coreshoptaxation/pimcore/css/taxation.css')->end()
                         ->end()
                     ->end()
-                    ->scalarNode('permissions')
-                        ->cannotBeOverwritten()
-                        ->defaultValue(['tax_rate', 'tax_rule_group'])
+                    ->arrayNode('permissions')
```

Comment:
don't overwrite permission configurations, you cannot change them anyway and its not part of this PR